apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/conf/Config.java

```diff
@@ -93,6 +93,11 @@
          * How depth the agent goes, when log cause exceptions.
          */
         public static int CAUSE_EXCEPTION_DEPTH = 5;
+
+        /**
+         * Peer max length, Used in the interception of Jedis, Redission, and Lettuce cluster configuration.
+         */
+        public static int PEER_MAX_LENGTH = 200;
```

Comment:
Move this to Plugin, this is not a part of core API, right? And don't put plugin name in comments, you wouldn't know when someone add it to somewhere else, considering `PeerUtil` is a core API

Reply:
I modified the comments and MD documentation
Moved the `PEER_MAX_LENGTH ` configured location
Modified the name of the class and changed to the enum tool class
Please review code again